plugin/LanguageClient.vim

```diff
@@ -98,6 +98,10 @@ function! LanguageClient_serverStatusMessage(...)
     return call('LanguageClient#serverStatusMessage', a:000)
 endfunction
 
+function! LanguageClient_isServerRunning(...)
+    return call('LanguageClient#isServerRunning', a:000)
```

Comment:
Where is this call being handled?

Reply:
The function itself is in the autoload directory, and I'm going to handle it there. I did the same for the other functions, but is there any problem?